js/renovation/ui/scheduler/workspaces/base/date_table/all_day_panel/layout.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  Component, ComponentBindings, JSXComponent, OneWay, Fragment,
+} from 'devextreme-generator/component_declaration/common';
+import { Table } from '../../table';
+import { AllDayPanelTableBody as TableBody } from './table_body';
+import { addHeightToStyle } from '../../../utils';
+import { GroupedViewData, ViewCellData } from '../../../types.d';
+import { AllDayPanelTitle as Title } from './title';
+
+export const viewFunction = (viewModel: AllDayPanelLayout) => (
+  <Fragment>
+    {viewModel.props.visible && (
+    <div
+      // eslint-disable-next-line react/jsx-props-no-spreading
+      {...viewModel.restAttributes}
+      className={`dx-scheduler-all-day-panel ${viewModel.props.className}`}
+      style={viewModel.style}
+    >
+      <Title />
+      <div>
+        <Table className="dx-scheduler-all-day-table">
+          <TableBody viewData={viewModel.allDayPanelData} />
+        </Table>
+      </div>
+    </div>
+    )}
+  </Fragment>
+);
+
+@ComponentBindings()
+export class AllDayPanelLayoutProps {
+  @OneWay() className?: string = '';
+
+  @OneWay() viewData?: GroupedViewData;
```

Comment:
This property already exists in `LayoutProps`. I suggest to extend them to avoid duplication